integraality/tests/test_app.py

```diff
@@ -80,14 +80,18 @@ class QueriesTests(PagesProcessorTests):
 
     def setUp(self):
         super().setUp()
-        patcher = patch('pages_processor.PropertyStatistics', autospec=True)
-        self.mock_property_statistics = patcher.start()
-        self.addCleanup(patcher.stop)
+        patcher1 = patch('pages_processor.PropertyStatistics', autospec=True)
+        self.mock_property_statistics = patcher1.start()
+        self.addCleanup(patcher1.stop)
+        patcher2 = patch('pages_processor.PropertyStatistics.GROUP_MAPPING', autospec=True)
+        self.mock_group_mapping = patcher2.start()
+        self.addCleanup(patcher2.stop)
```

Comment:
I’m not very happy I had to do that mocking, for no good reason except for the tests to pass (well, I guess that is a good reason in itself :p)

Reply:
> (The only thing is that I don’t explicitly use it in make_footer − because there is 'no grouping'. Should I output something anyways? And then, should we keep using the empty string, or something more meaningful/explicit?

I think its fine as is. Skipping both the param and the value makes sense to me. If we still want to be super explicit then `"any"` is probably the most suitable string.